packages/commercetools/api-client/src/index.ts

```diff
@@ -29,18 +29,23 @@ const setup = <TCacheShape>(setupConfig?: SetupConfig<TCacheShape>): ApolloClien
     ...setupConfig.customOptions
   })
   locale = setupConfig.locale
-  currency = setupConfig.currency
+  currency = (setupConfig.currency || '').toUpperCase()
   country = setupConfig.country
 
   return apolloClient
 }
 
+const updateStoreConfig = (storeConfig: StoreConfig) => {
```

Comment:
imho we should save config passed to `setup` method in `setup` variable and make it exportable. Then we can easily update it's certain properties without unnecessary abstractions like this method.